CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 
 **Internal**:
 
+- Remove transaction metrics allowlist. ([#2092](https://github.com/getsentry/relay/pull/2092))
```

Comment:
Should we add this to a `breaking changes` section in the changelog?

Reply:
I asked Jan and got a no